<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-4HJ5QRS" failures="1" tests="3" name="Test" time="34.724" errors="0" timestamp="2024-04-30T12:52:46 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="12.207">
    <skipped/>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="10.337">
    <skipped/>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="10.604">
    <failure type="java.lang.AssertionError" message="Login Successfully is not matching with expected : Login expected [Login] but found [Login Successfully]">
      <![CDATA[java.lang.AssertionError: Login Successfully is not matching with expected : Login expected [Login] but found [Login Successfully]
at definitions.sampledefinition.verify_the_home_page(sampledefinition.java:154)
at âœ½.Verify the Home Page(file:///C:/Users/hp/eclipse-workspace/MyProject/src/test/resources/features/VerifyHomePage.feature:9)
... Removed 4 stack frames]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Test -->
